test/rpc_test.py

```diff
@@ -265,10 +265,13 @@ def test_register_rpc_backend_and_start_rpc_backend(
     @dist_init(setup_model_parallel=False)
     def test_duplicate_name(self):
         with self.assertRaisesRegex(RuntimeError, "is not unique"):
-            rpc.init_model_parallel(
-                self_name="duplicate_name",
+            store, _, _ = next(torch.distributed.rendezvous(
```

Comment:
why calling the lower level APIs instead of `init_model_parallel`?

Reply:
With the current changes `init_model_parallel` throws a KeyError because the worker_name_to_id[self_name] lookup fails for invalid strings. I call the lower level API here to reproduce the errors we're looking for.

Reply:
I see, we will need to update these tests after we move `worker_name_to_id` to `backend_config` dict.